packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.tsx

```diff
@@ -95,6 +100,10 @@ export class ExpandingCard extends BaseComponent<IExpandingCardProps, IExpanding
     );
   }
 
+  public get element(): HTMLDivElement {
```

Comment:
I know this is an anti-pattern. I had to expose it to use in `HoverCard` for measurement. Is there a better way to get hold of the element?